application/src/main/resources/thingsboard-mqtt-broker.yml

```diff
@@ -821,25 +794,20 @@ cache:
   # The common prefix for all cache keys. Defaults to empty string meaning no prefix is added
   cache-prefix: "${CACHE_PREFIX:}"
   specs:
-    packetIdAndSerialNumber:
-      # Cache TTL in minutes
-      timeToLiveInMinutes: "${CACHE_SPECS_PACKET_ID_SERIAL_NUMBER_TTL:1440}"
-      # Cache max size. 0 means the cache is disabled in case CACHE_TYPE=caffeine
-      maxSize: "${CACHE_SPECS_PACKET_ID_SERIAL_NUMBER_MAX_SIZE:10000}"
     mqttClientCredentials:
       # Cache TTL in minutes
       timeToLiveInMinutes: "${CACHE_SPECS_MQTT_CLIENT_CREDENTIALS_TTL:1440}"
-      # Cache max size. 0 means the cache is disabled in case CACHE_TYPE=caffeine
+      # Cache max size.
```

Comment:
pls remove dots from the end of each line :)
All the comments do not have dots at the end. Only if there are several sentences in the comment then there are dots to separate them. Sorry for my idealization

Reply:
NP. I appreciate well-defined rules and consistency in formatting. This makes things clearer.